src/lib/Event/Subscriber/TrashEventSubscriber.php

```diff
@@ -68,8 +90,9 @@ public function onTrash(TrashEvent $event): void
      */
     public function getRelations(ContentInfo $contentInfo): array
     {
-        return $this->contentService
-            ->loadReverseRelations($contentInfo);
+        return $this->repository->sudo(function () use ($contentInfo) {
```

Comment:
This usage needs a comments explaining the reasoning why it was used like that. Either directly above the code (prefered, as it's an implementation detail) or in the method description.

Also, missing return type.

```suggestion
        return $this->repository->sudo(function () use ($contentInfo): array {
```

Reply:
Still not what I was hoping for :smile:. You explain it well in PR description though.

```
Sudo must be used to have access to trash and load content relations
```

As it currently stands you explain what `sudo` is for (to access trash and related content without permission checks). However this doesn't add value - when looking at this code after a few months I would still be wondering "why is it there"? What exact issue this solves? How this method behaves if the `sudo` is not there?

```
Sudo must be used to have access to trash and load content relations, since Client using this EventSubscriber operates as a User without privileges.
```